packages/aragon-wrapper/src/cache/index.js

```diff
@@ -10,57 +8,35 @@ export default class Cache {
   constructor (prefix) {
     this.prefix = prefix
 
-    // Set up cache DB
-    let adapter = null
-    if (typeof window === 'undefined') {
-      // TODO: Support caching on the file system
-      // const path = require('path')
-      // const os = require('os')
-      // const FileAsync = require('lowdb/adapters/FileAsync')
-
-      // adapter = new FileAsync(
-      //   path.resolve(
-      //     os.homedir(), '.aragon', 'cache.json'
-      //   )
-      // )
-      adapter = new Memory()
-    } else {
-      adapter = new LocalStorage()
-    }
-    this.db = low(adapter)
-
-    // Set default cache state
-    this.db.defaults({}).write()
+    // if (typeof window === 'undefined') {
+    //   // TODO: Support caching on the file system
+    //   const path = require('path')
+    //   const os = require('os')
+    // }
 
     // Set up the changes observable
     this.changes = new Subject()
-  }
 
-  getCacheKeyPath (key) {
-    return `${this.prefix}.${key}`
+    // Set up cache DB
+    this.db = localforage.createInstance({
+      driver: [localforage.INDEXEDDB, localforage.LOCALSTORAGE],
+      name: `localforage/${this.prefix}`
+    })
   }
 
   async set (key, value) {
-    // Some lowdb adapters are synchronous while others are asynchronous so
-    // let's always wrap it in a promise
-    await Promise.resolve(this.db.set(
-      this.getCacheKeyPath(key),
+    await this.db.setItem(
+      key,
       value
-    ).write())
+    )
 
     this.changes.next({ key, value })
   }
 
-  get (key, defaultValue) {
-    return this.db.get(
-      this.getCacheKeyPath(key)
-    ).value() || defaultValue
-  }
-
-  update (key, transition) {
```

Comment:
I can't see this being used anywhere, I think it's safe to say this is not a breaking change.
What do you think @sohkai ?

Reply:
Sounds good to me! The breaking change won't be a problem for now anyway as we're still on beta versions :).